recipe/0118_multivalue/manifest.json

```diff
@@ -33,25 +33,25 @@
     },
     "items": [
       {
-        "id": "https://example.org/iiif/text-language/canvas1",
+        "id": "https://iiif.io/api/cookbook/recipe/0118_multivalue/canvas1",
```

Comment:
Could you use "{{ id.path }}/canvas/1" again so it builds on both preview and iiif.io